source/system-administrators/engine/configure-engine-multi-tenancy.rst

```diff
@@ -33,6 +33,11 @@ The root folder path, as shown below, needs to be configured to include a substi
 This variable will be resolved by Crafter Engine for each request. There are two possible
 configurations to resolve this value.
 
+- simple multi-tenancy
+- mapped multi-tenancy
+
+We recommend using simple multi-tenancy with a web server like Apache or a CDN in front of Crafter Engine
```

Comment:
I think we should change this to: `Simple multi-tenancy is the recommended configuration in most cases, with an Apache HTTP server, NGINX, or CDN proxying Crafter Engine.`